src/components/ButtonRadio/index.tsx

```diff
@@ -0,0 +1,53 @@
+import clsx from "clsx";
+import { useEffect, useState } from "react";
+
+type IButtonRadioOption = {
```

Comment:
if it is a type then you do not prefix with I

Reply:
Updated